pandas/tests/extension/base/ops.py

```diff
@@ -51,33 +51,39 @@ def _check_divmod_op(self, s, op, other, exc=NotImplementedError):
 
 class BaseArithmeticOpsTests(BaseOpsUtil):
     """Various Series and DataFrame arithmetic ops methods."""
+    series_scalar_exc = TypeError
+    frame_scalar_exc = TypeError
+    series_array_exc = TypeError
+    divmod_exc = TypeError
 
     def test_arith_series_with_scalar(self, data, all_arithmetic_operators):
         # series & scalar
         op_name = all_arithmetic_operators
         s = pd.Series(data)
-        self.check_opname(s, op_name, s.iloc[0], exc=TypeError)
+        self.check_opname(s, op_name, s.iloc[0], exc=self.series_scalar_exc)
 
     @pytest.mark.xfail(run=False, reason="_reduce needs implementation")
     def test_arith_frame_with_scalar(self, data, all_arithmetic_operators):
         # frame & scalar
         op_name = all_arithmetic_operators
         df = pd.DataFrame({'A': data})
-        self.check_opname(df, op_name, data[0], exc=TypeError)
+        self.check_opname(df, op_name, data[0], exc=self.frame_scalar_exc)
 
     def test_arith_series_with_array(self, data, all_arithmetic_operators):
         # ndarray & other series
         op_name = all_arithmetic_operators
         s = pd.Series(data)
-        self.check_opname(s, op_name, [s.iloc[0]] * len(s), exc=TypeError)
+        self.check_opname(s, op_name, pd.Series([s.iloc[0]] * len(s)),
+                          exc=self.series_array_exc)
 
     def test_divmod(self, data):
         s = pd.Series(data)
-        self._check_divmod_op(s, divmod, 1, exc=TypeError)
-        self._check_divmod_op(1, ops.rdivmod, s, exc=TypeError)
+        self._check_divmod_op(s, divmod, 1, exc=self.divmod_exc)
+        self._check_divmod_op(1, ops.rdivmod, s, exc=self.divmod_exc)
 
     def test_error(self, data, all_arithmetic_operators):
         # invalid ops
+        # What is this testing?
```

Comment:
I think this was used to test classes that simply error for ops, so if you don't have this, you basically have to override or skip this. Looking at it now, maybe not very useful/handy to have, as actually the above tests already can test an op errors